api/next.pb.txt

```diff
@@ -762,6 +770,34 @@ file {
       type: TYPE_BYTES
       json_name: "data"
     }
+    field {
```

Comment:
@dmcgowan Any reason these weren't a part of the previous PR? Did we have a bad merge? Basically, it looks like we merged around the same time and missed fields.

@cpuguy83 I'm going to update these in a separate PR.